doc/question-templates.md

```diff
@@ -2,9 +2,67 @@
 
 Question templates live in `lib/smart_answer_flows/<flow-name>/questions/<question-name>.govspeak.erb`.
 
-The templates can contain content for the `title`, `hint`, `suffix_label`, `body` and `post_body`, all of which are optional.
+## Content types
 
-The templates can contain the options for multiple choice questions.
+The templates can contain content for any of the following keys:
+
+### `title(text)`
+
+* Valid for all question types
+* `text` argument is a String
+* Used as the heading (currently "h2")
+
+### `options(hash)`
+
+* Valid for multiple choice & checkbox question types
+* `hash` argument is a `Hash` of option key Strings versus option text Strings
+* Used to "translate" options keys for multiple choice & checkbox questions into human-friendly text
+
+### `hint(text)`
+
+* Valid for all question types
+* `text` argument is a String
+* Used as a "hint" paragraph
+* Also used to generate a label for value & postcode questions
+
+> I'm not convinced the `hint` key should be overloaded by the value & postcode question types - the `label` key would seem more appropriate.
+
+### `label(text)`
+
+* Valid for value questions
+* `text` argument is a String
+* Used as a label (preceding the input control)
+
+### `suffix_label(text)`
+
+* Valid for value & money questions
+* `text` argument is a String
+* Used as the label (following the input control)
+
+### `body(govspeak)`
+
+* Valid for all question types
+* `govspeak` argument is a String in [Govspeak][] format
+* Used to generate the main content (appearing above the input control)
```

Comment:
Change: (appearing above the input control)
To: (appearing above the input control, e.g. the start button)